packages/grafana-ui/src/components/Icon/Icon.tsx

```diff
@@ -1,14 +1,104 @@
-import React, { ComponentType } from 'react';
+import React from 'react';
 import { css, cx } from 'emotion';
-import { GrafanaTheme, toPascalCase } from '@grafana/data';
+import { GrafanaTheme } from '@grafana/data';
 import { stylesFactory } from '../../themes/stylesFactory';
 import { useTheme } from '../../themes/ThemeContext';
 import { IconName, IconType, IconSize } from '../../types/icon';
-//@ts-ignore
-import * as DefaultIcon from '@iconscout/react-unicons';
-import * as MonoIcon from './assets';
-import { customIcons } from './custom';
-import { SvgProps } from './assets/types';
+import SVG, { cacheStore } from '@leeoniya/react-inlinesvg';
+
+// inlined static cache
+
+import i001 from '!!raw-loader!../../../../../public/img/icons/unicons/bars.svg';
```

Comment:
Nit: move to iconsCache.ts next to Icon.tsx

Reply:
I moved this to a file named iconBundle.ts and I also changed so this is directly imported from app.ts so that it's not part to any @grafana/ui bundle as the icons are really in the app anyway.